evm/src/cpu/kernel/asm/mpt/util.asm

```diff
@@ -10,6 +10,17 @@
     // stack: (empty)
 %endmacro
 
+%macro initialize_rlp_segment
+    PUSH 0x80
+    // We need to alloc a two rlp blocks because first positions will be used for storing the txn rlp.
+    // The encoding of the empty node will be stored at ENCODED_EMPTY_NODE_POS = 0x10009
+    %alloc_rlp_block
+    POP
+    %alloc_rlp_block
+global debug_position:
```

Comment:
this is fine because it is called only once, but you shouldn't ever have `global` labels within macros, so I'd suggest removing it.